crates/ruff_python_ast/src/nodes.rs

```diff
@@ -927,6 +927,12 @@ impl From<ExprJoinedStr> for Expr {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum JoinedStringPart {
+    String(StringTodoName),
```

Comment:
Maybe `String(StringTodoName)` -> `Constant(StringPart)` or `Constant(ConstantStringPart)`

Reply:
My preference (in the long term) is to rename `Constant` (it's a constant at runtime but it is literal in the source code) to `Literal`. How about `FStringLiteralElement` and `FStringExpressionElement` (or `FormattedValueElement`).